guides/common/modules/proc_using-container-registries.adoc

```diff
@@ -30,3 +33,39 @@ Pulling container images:
 ----
 # podman pull {foreman-example-com}/my-image:<optional_tag>
 ----
+
+Pushing container images to the {Project} container registry:
+
+* To indicate which organization, product, and repository the container image belongs to, use container namespaces.
+* All labels are composed of entirely lowercase characters regardless of actual label capitalization.
+* Pushes follow one of two formats:
+[options="nowrap", subs="+quotes,attributes"]
+----
+# podman push <image> example.com/<organization_label>/<product_label>/<repository_name>[:tag]
+# podman push <image> example.com/id/<organization_id>/<product_id>/<repository_name>[:tag]
+----
+
+* A complete push looks like the following:
+[options="nowrap", subs="+quotes,attributes"]
+----
+# podman push d2c94e258dcb example.com/default_organization/test_product/hello-world:latest
+Getting image source signatures
+Copying blob ac28800ec8bb done   |
+Copying config d2c94e258d done   |
+Writing manifest to image destination
+----
+
+.Limitations on container registries
```

Comment:
Can you move this subheading for it to be placed somewhere closer to the beginning of the module? I don't think the information here will be relevant after `.Procedure`, after users have run the command(s).

Reply:
Also, feel free to reevaluate whether including the limitations as bullet points still works if/after you implement my suggestions. It's quite possible that after the edits, regular paragraphs will make more sense.